example/ios/xmtpreactnativesdkexample.xcodeproj/project.pbxproj

```diff
@@ -424,7 +424,7 @@
 					"-lc++",
 				);
 				OTHER_SWIFT_FLAGS = "$(inherited) -D EXPO_CONFIGURATION_DEBUG";
-				PRODUCT_BUNDLE_IDENTIFIER = expo.modules.xmtpreactnativesdk.example;
+				PRODUCT_BUNDLE_IDENTIFIER = cvoell.modules.xmtpreactnativesdk.example;
```

Comment:
Can probably drop the changes on this file

Reply:
ah accidentally picked that up, thanks for catching

Reply:
updated 3bf90550294e5ed6f7876fed4fef046661647431